### PR TITLE
Numpy frontend nanstd #7071

### PR DESCRIPTION
**Pull request by JortdeJong13**
My previous pull request for this issue seems to be gone, so I'm making this new one.
I implemented a test function for _nanstd_, but I can't run the tests because of a code refractor that makes all tests fail. I used the other test functions in _ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py_ as reference.

### CODE REVIEW

**Comment by JortdeJong13** [comment]
Close #7071

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py [hunk 1]
```diff
@@ -182,3 +182,60 @@ def test_numpy_std(
...
+)
+def test_numpy_nanstd(
+    dtype_and_a,
+    dtype,
+    where,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    fw,
```
Comment (jkeane508) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py:206]: You no longer need to specify this.

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py [hunk 2]
```diff
@@ -182,3 +182,60 @@ def test_numpy_std(
...
+        native_array=native_array,
+    )
+
+    np_frontend_helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
```
Comment (jkeane508) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py:230]: or this

**Review by jkeane508 — CHANGES_REQUESTED** [review]
After chatting with the testing team, it seems that you no longer need to specify the framework for frontend tests.

**Comment by JortdeJong13** [comment]
I made the changes as requested. Are the tests still broken? All the tests in _test_averages_and_variances.py_ seem to fail.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by JortdeJong13** [comment]
.

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Hey, sorry this PR completely slipped my radar. The files seem to have lint errors, you can fix this by installing a pre-commit hook as documented [here](https://lets-unify.ai/ivy/contributing/setting_up.html#pre-commit).

It also seems as though the tests didn't run properly, will follow up once I can see the test logs :)

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Hey,

It seems the tests are still not running correctly, can you update your branch? Perhaps that would fix some bugs that have since been resolved. :)

Thanks!

**Comment by JortdeJong13** [comment]
The existing tests in _ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py_ still seem bugged.

**Review by jkeane508 — APPROVED** [review]
lgtm :)

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Now that Master is open again, just need to resolve conflicts and we can go ahead and merge to master :)

**Comment by JortdeJong13** [comment]
I resolved the conflicts.

**Pull request merged**